hphp/runtime/base/runtime-option.cpp

```diff
@@ -487,6 +492,14 @@ static inline bool alignMacroFusionPairs() {
   return false;
 }
 
+static inline bool armLseDefault() {
+#ifdef __linux__
+  return (getauxval(AT_HWCAP) & HWCAP_ATOMICS) != 0;
```

Comment:
Actually I think the #ifdef should be:
#if defined (__linux__) && defined (__aarch64__)
Because HWCAP_ATOMICS is not defined on all targets.